geoips/plugins/modules/procflows/config_based.py

```diff
@@ -764,21 +791,36 @@ def call(fnames, command_line_args=None):
         fnames = glob(config_dict["filenames"])
 
     output_file_list_fname = command_line_args["output_file_list_fname"]
+    reader_kwargs = None
     bg_files = None
     bg_product_name = None
+    bg_reader_kwargs = None
     bg_resampled_read = False
     bg_self_register_dataset = None
     bg_self_register_source = None
 
+    if command_line_args.get("reader_kwargs"):
+        reader_kwargs = command_line_args["reader_kwargs"]
+    elif "reader_kwargs" in config_dict:
+        reader_kwargs = config_dict.get("reader_kwargs")
+    if not reader_kwargs:
+        reader_kwargs = {}
+
     if command_line_args.get("fuse_files") is not None:
         bg_files = command_line_args["fuse_files"][0]
     elif "fuse_files" in config_dict:
         bg_files = glob(config_dict["fuse_files"])
 
     if command_line_args.get("fuse_reader") is not None:
-        bg_reader = readers.get_plugin(command_line_args["fuse_reader"][0])
+        bg_reader_plugin = readers.get_plugin(command_line_args["fuse_reader"][0])
```

Comment:
Also update naming to "bg_reader_plugin" for consistency.  All calls to bg_reader_plugin should have **bg_reader_kwargs added to the call.